languages/kotlin/src/main/cpp/jni_view_node.cpp

```
@@ -36,6 +36,87 @@ JNIEXPORT void JNICALL Java_com_github_kavos113_karin_engine_jni_JniViewNode_set
     );
 }
 
+JNIEXPORT void JNICALL Java_com_github_kavos113_karin_engine_jni_JniViewNode_setSize
+    (JNIEnv *env, jclass cls, jlong viewPtr, jfloat width, jfloat height)
+{
+    auto *node = reinterpret_cast<ViewNode *>(viewPtr);
+    node->setSize({width, height});
```

Cppcheck (2.20.0)

[error] Found an exit path from function with non-void return type that has missing return statement

(missingReturn)